### PR TITLE
Life Insurance SEO Optimization

### PR DESCRIPTION
**Pull request by lillianjwarner**
## Page to edit
url: Life insurance

## Origin of request (internal/stakeholder/user feedback)
SEO 

## Description of what's needed (edits/link changes/additions)
Life Insurance section

Adding browser titles
Updating intro paragraphs
Updating additional SEO opportunities as relevant

### CODE REVIEW

**Comment by peggygannon** [comment]
@ncksllvn Hi, Nick, can you take a look at this PR. there were several conflicts (which I resolved), but I would appreciate your eye on the code. If all looks good, then we can merge! thank you.

File: pages/life-insurance/totally-disabled-or-terminally-ill.md [hunk 1]
```diff
@@ -11,17 +14,24 @@ relatedlinks:
     links:
     - url: /life-insurance/manage-your-policy/
       title: Access your policy online
-      description: Need to file a benefits claim after your family member has died? Access your policy online.
-
+      description: Need to file a benefits claim? Access your policy online.
+aliases:
+  - /life-insurance/disabled-and-terminally-ill/
```
Comment (ncksllvn) [inline pages/life-insurance/totally-disabled-or-terminally-ill.md]: `aliases` is actually not supported anymore. Do we need a server-side redirect for this?

Comment (peggygannon) [reply pages/life-insurance/totally-disabled-or-terminally-ill.md]: @ncksllvn thanks. Yes, we should do a server side redirect.

 I can remove this alias line from the file.

Comment (ncksllvn) [reply pages/life-insurance/totally-disabled-or-terminally-ill.md]: Thanks! If you can, please write a ticket for the server-side redirect and assign me!

Comment (peggygannon) [reply pages/life-insurance/totally-disabled-or-terminally-ill.md]: @ncksllvn ok, will do!

**Pull request merged**